ci/test-checks.sh

```diff
@@ -25,7 +25,7 @@ fi
 _ cargo +"$rust_stable" fmt --all -- --check
 
 _ cargo +"$rust_stable" clippy --version
-_ cargo +"$rust_stable" clippy --workspace -- --deny=warnings
+_ cargo +"$rust_stable" clippy --workspace --tests -- --deny=warnings
```

Comment:
@svenski123 Btw, I'd like to add `--all-targets` (this includes `--benches`), but currently I can't. Interested? :)

Reply:
Have a look at #10445